desktop.bundles/index/index.bemjson.js

```diff
@@ -4,8 +4,9 @@ module.exports = {
     favicon: '/favicon.ico',
     head: [
         { elem: 'meta', attrs: { name: 'description', content: '' } },
-        { elem: 'meta', attrs: { name: 'viewport', content: 'width=device-width, initial-scale=1' } },
-        { elem: 'css', url: 'index.min.css' }
+        { elem: 'meta', attrs: { name: 'viewport', content: 'width=device-width, initial-scale=1, initial-scale=1.0, minimum-scale=1.0, maximum-scale=1.0, user-scalable=no' } },
```

Comment:
`initial-scale=1, initial-scale=1.0` at the same time ;)

use just `1` without zero.